libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4608,10 +4611,10 @@ void GCS_MAVLINK::send_sys_status()
 
     const AP_BattMonitor &battery = AP::battery();
     float battery_current;
-    int8_t battery_remaining;
+    uint8_t battery_remaining = -1;
+    IGNORE_RETURN(battery.capacity_remaining_pct(battery_remaining));
```

Comment:
Shouldn't this remain within the `healthy()` check?

i.e. if the battery isn't healthy we may have stopped receiving updates from it in which case the percentage may be stale.

Similarly below

Reply:
Yep, you're right! fixed it.